tests/test_bundle_download.py

```diff
@@ -58,6 +58,12 @@
     "model.ts",
 ]
 
+TEST_CASE_6 = [
+    ["models/model.pt", "models/model.ts", "configs/train.json"],
+    "brats_mri_segmentation",
```

Comment:
Is it necessary to upload a fake model to monaihosting because the real one here may download more slowly?

Reply:
may include a minimal example from https://github.com/ericspod/tutorials/blob/bundles_update/bundle/bundle_intro.ipynb once that's completed